prep-css-layout-classes/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Layout Classes</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+    /* stylelint-disable block-no-empty */
+    body {
+      font-family: sans-serif;
+    }
+.box{
+  display: flex;
+  /* justify-content: center; */
+}
+h1, h3{
+text-align: center;
+}
+.box{
+  width:100%;
+}
+.box_item{
+      width: 30%;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+}
+.box_item div{
+      width: 60%;
+}
```

Comment:
For this exercise we want to create some layout classes, `.container`, `.row`, and `.column`. These will be used to divide the page into a three-column layout.

<img width="893" alt="image" src="https://user-images.githubusercontent.com/7432943/194124111-3b835e9f-1c0a-4ce3-97a7-149ba88301f0.png">